src/main/java/com/toofifty/easyblastfurnace/utils/Strings.java

```diff
@@ -42,7 +42,7 @@ public class Strings {
     public static String STEEL = "Steel bars";
 
     // Patterns
-    public static String COAL_FULL = "^The coal bag is full.$";
```

Comment:
This is better left as "is full", those with 99 smithing and wearing a smithing cape can put 36 coal in their coal bag.

Reply:
I think "is full" is wrong for both people with and without smithing cape. In this video from the iron skilling discord https://streamable.com/tyowzu , it looks like the coal bag just says "contains 36 pieces of coal".

Reply:
It seems you're right, good spot. Now you've pointed it out, I don't think that the particular bit of code here has ever worked, so actually it would be safe to leave it for now.

The coal bag filling is covered by a menu event check in `com/toofifty/easyblastfurnace/EasyBlastFurnacePlugin.java:239`. Basically the bag gets updated whenever you click "Fill", so yeah. The bit of code you've highlighted is broken, but thankfully not in a function-breaking way.

You have however given me some ideas on how to fix some annoying edge cases that come up with the coal bag every now and then due to the menu entity checks sometimes over-zealously counting clicks. Maybe it's for another PR though, I've got to think about it a little bit.